buildSrc/build.gradle

```diff
@@ -17,6 +17,7 @@
  * under the License.
  */
 import java.nio.file.Files
+import org.gradle.util.GradleVersion
```

Comment:
Did something change in how gradle does imports of gradle classes that requires explicitly adding this import?

Reply:
Yes, relying on explicit imports is deprecated and triggers a deprecation warning.